### PR TITLE
Wording around resource-ids used in PUT vs POST

### PR DESCRIPTION
**Pull request by tfrauenstein**
Minor editorial update for more clarity.

### CODE REVIEW

File: chapters/http-requests.adoc [hunk 1]
```diff
@@ -124,16 +124,13 @@ been updated), {201} (if resources have been created), {202} (if the request
...
 {Location} header (if the actual resource is not returned).
 
-The semantic for single resource endpoints is best described as _"please
-execute the given well specified request on the resource identified by the
-URL"_.
-
-*Generally:* {POST} should be used for scenarios that cannot be covered by the
-other methods sufficiently. In such cases, make sure to document the fact that
-{POST} is used as a workaround (see {GET-with-Body}). 
+Note, by using POST to create resources the resource ID must not be passed as
```
Comment (tkrop) [inline chapters/http-requests.adoc:127]: `Note`-> `*Note:*`?
`POST` -> `{POST}`!

File: chapters/http-requests.adoc [hunk 2]
```diff
@@ -124,16 +124,13 @@ been updated), {201} (if resources have been created), {202} (if the request
...
-{POST} is used as a workaround (see {GET-with-Body}). 
+Note, by using POST to create resources the resource ID must not be passed as 
+request input date by the client, but created and maintained by the service and 
+returned with the response payload. 
 
-*Note:* Resource IDs with respect to {POST} requests are created and maintained
-by server and returned with response payload.
+Apart from resource creation, {POST} should be also used for scenarios that cannot 
+be covered by the other methods sufficiently. However, in such cases make sure to 
+document the fact that {POST} is used as a workaround (see {GET-with-Body}).
```
Comment (tkrop) [inline chapters/http-requests.adoc:133]: shouldn't we write here `(see e.g. {GET-with-Body})` since it is only an example.

**Review by tkrop — APPROVED** [review]

File: chapters/http-requests.adoc (hunk 1 above)
Comment (tfrauenstein) [reply chapters/http-requests.adoc:127]: done

File: chapters/http-requests.adoc (hunk 2 above)
Comment (tfrauenstein) [reply chapters/http-requests.adoc:133]: done

**Comment by tkrop** [comment]
:+1:

**Comment by vadeg** [comment]
👍

**Pull request merged**